plugins/default-browser-emulator/injected-scripts/SharedWorker.prototype.ts

```diff
@@ -1,26 +1,52 @@
+export type Args = never;
+
 if (typeof SharedWorker === 'undefined') {
   // @ts-ignore
   return;
 }
 
+const OriginalSharedWorker = SharedWorker;
+const originalSharedWorkerProperties = ObjectCached.getOwnPropertyDescriptors(SharedWorker);
+
 // shared workers created from blobs don't automatically pause in devtools, so we have to manipulate
-proxyConstructor(self, 'SharedWorker', (target, argArray) => {
-  if (!argArray?.length) return ReflectCached.construct(target, argArray);
+ObjectCached.defineProperty(self, 'SharedWorker', {
+  // eslint-disable-next-line object-shorthand
+  value: function SharedWorker(this, scriptURL, options) {
+    // eslint-disable-next-line strict
+    'use strict';
```

Comment:
Should we add a comment here about why we're activating strict mode and then checking new.target?